### PR TITLE
[SDL-0273] Add HMI status test scripts for WEB_VIEW applications

### PR DESCRIPTION
**Pull request by aderiabin**
Jira: https://adc.luxoft.com/jira/browse/FORDTCN-6817

This PR is **ready** for review.

### Summary
Add HMI status test scripts for WEB_VIEW applications

### ATF version
develop

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+--[[ Local Constants ]]
+local APP = 1
+local WEB_ENGINE = 1
+
+--[[ Local Variables ]]
+local testCases = {
+  [001] = { t = "WEB_VIEW", m = true,  s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" }
```
Comment (YarikMamykin) [inline test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua:37]: Propose to change 
```suggestion
    [1] = { action = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" }
```
This relates to other scripts.

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+
+--[[ Local Constants ]]
+local APP = 1
+local WEB_ENGINE = 1
+
+--[[ Local Variables ]]
+local testCases = {
+  [001] = { t = "WEB_VIEW", m = true,  s = {
```
Comment (YarikMamykin) [inline test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua:36]: What is `t`, `m` and `s`? Maybe it is better to use full name of param?
This relates to other scripts.

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "AUDIBLE",     vSS = "NOT_STREAMABLE" }
+  }},
+  [002] = { t = "WEB_VIEW", m = false, s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" }
```
Comment (LitvinenkoIra) [inline test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua:40]:
```suggestion
    [1] = { action = common.userActions.activateApp,   HMILvl = "FULL",       audioSS = "NOT_AUDIBLE", videoSS = "NOT_STREAMABLE" }
```

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+  }},
+  [017] = { t = "WEB_VIEW", m = false, s = {
+    [1] = { e = common.userActions.activateApp,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [2] = { e = common.userActions.deactivateHMI, hLvl = "BACKGROUND", aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" },
+    [3] = { e = common.userActions.activateHMI,   hLvl = "FULL",       aSS = "NOT_AUDIBLE", vSS = "NOT_STREAMABLE" }
+  }}
+}
+
+--[[ Local Functions ]]
+local function doAction(pSS)
```
Comment (LitvinenkoIra) [inline test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua:114]: @aderiabin `pSS` not obvious parameter name

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua (hunk 1 above)
Comment (aderiabin) [reply test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua]: @YarikMamykin Done (see 77f3211 )

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua (hunk 2 above)
Comment (aderiabin) [reply test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua]: @YarikMamykin Done (see 77f3211 )

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua (hunk 3 above)
Comment (aderiabin) [reply test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua]: @LitvinenkoIra Done (see 77f3211 )

File: test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua (hunk 4 above)
Comment (aderiabin) [reply test_scripts/WebEngine/WebEngineProjection/020_HMI_status_transitions.lua]: @LitvinenkoIra Done (see 77f3211)

**Pull request merged**